src/modules/tools/temperaturecontrol/predefined_thermistors.h

```diff
@@ -23,10 +24,13 @@ typedef struct {
         float c1, c2, c3;
 } const thermistor_table_t;
 
-// Using https://github.com/MarlinFirmware/Marlin/blob/Development/Marlin/scripts/createTemperatureLookupMarlin.py to find the coefficeints for these
+// Use one of the following scripts to calcuate the coefficients:
+// - http://www.thinksrs.com/downloads/programs/Therm%20Calc/NTCCalibrator/NTCcalculator.htm
+// - https://github.com/MarlinFirmware/Marlin/blob/Development/Marlin/scripts/createTemperatureLookupMarlin.py
 static const thermistor_table_t predefined_thermistors[] {
     // name,            r1,  r2,   c1,                    c2,                     c3
     {"EPCOS100K",       0,   4700, 0.000722378300319346F, 0.000216301852054578F,  9.2641025635702e-08F},  // B57540G0104F000
+    {"Vishay100K"       0,   4700, 0.0007022607370F,      0.0002209155484F,       7.101626461e-08F    },  // NTCS0603E3104FXT
```

Comment:
Missing , please do not submit requests you have not tested or compiled. Thank you